### DISCUSSION TITLE
OpenLoginAdapter adapterSettings 'replaceUrlOnRedirect: false' is not working

### DISCUSSION THREAD

**Discussion by jisoo-3pm** [Web3Auth/Web3Auth#678] [Help]
I set replaceUrlOnRedirect to false to keep the query parameters, but it removes original query parameters.
Is this a bug?

**Comment by shahbaz17** [comment]
Hey @jisoo-3pm 

Can you provide more info here? Such as error screenshots, code snippets, SDKs used, etc.

**Comment by chaitanyapotti** [comment]
it should be working correctly.
please update to the latest version of sdk and try again. 
If not, please share code snippet here
